receiver/awscontainerinsightreceiver/internal/stores/utils_test.go

```diff
@@ -66,21 +66,25 @@ type mockNodeInfoProvider struct {
 func (m *mockNodeInfoProvider) NodeToCapacityMap() map[string]v1.ResourceList {
 	return map[string]v1.ResourceList{
```

Comment:
should add a test for non-gpu node?

Reply:
Updated `testNode2` to not have any GPUs and used that in the test to validate the value is returned as 0.
I was debating if the func should return false if it cant find gpu capacity set on it.
But current pattern seems to be to return false if it cant find the conditions map at all. So the 0 check should still indicate if the node has any GPUs or not.